berglas_python.py

```diff
@@ -100,8 +100,7 @@ def Resolve(project_id: str, env_var_value: str) -> str:
     """
 
     if not env_var_value.startswith(BERGLAS_PREFIX):
-        logging.info(f"No berglas prefix for the env var value {env_var_value}")
-        return
+        env_var_value = f'{BERGLAS_PREFIX}{env_var_value}'
```

Comment:
No, the decryption process will failed if the EnvVar not start by berglas://
User can chose to pass all its env var to this lib and, only the eligible ones are decrypted, the other are return as is.

Reply:
I corrected this [in this commit](https://github.com/guillaumeblaquiere/berglas-python/pull/1/commits/c2f4165c934b23f71772648dc4bb1b8691d61904)